Model/Api/Client.php

```diff
@@ -100,7 +100,7 @@ private function sendHttpRequest(TokenInterface $token, Request $request)
                 'method' => $request->getMethod(),
                 'header' => implode("\r\n", $this->prepareHeaders($token, $request)),
                 'content' => $isJson ? $this->json->serialize($request->getBody()) : $request->getBody(),
-                'ignore_errors' => false,
+                'ignore_errors' => true,
```

Comment:
why?
This change dosen't sound related to the PR (wysiwyg)

Reply:
@tmotyl when you try to submit not valid description the error is HTTP 422 status without any other message. To have more information what is wrong with your offer and handle error message this change is required.